9_report/src/fig_input_data.R

```diff
@@ -0,0 +1,48 @@
+fig_input_data <- function(fig_ind, preds_ind, remake_file, config_file){
+  #discharge, concentration, loads
+
+  site = '05465500'
+  model = 'long1'
+  ref_date = '2017-05-09'
+
+  preds <- readRDS(sc_retrieve(preds_ind, remake_file)) %>%
+    dplyr::filter(ref_date == '2017-05-09', # dplyr::filter doesn't recognize objects??
```

Comment:
objects? what kinds of object did you want it to recognize?

Reply:
the ones defined on lines 4-6 (ref_date, etc...). Or could replace with a yml file as input.

Reply:
`dplyr` by default uses non-standard evaluation (hence we can use `ref_date` and it knows we're talking about the column). However, there's been a lot of work recently  to make it friendlier to standard evaluation. This should work:
```
preds <- readRDS(sc_retrieve(preds_ind, remake_file)) %>%
     dplyr::filter(ref_date == !!(ref_date ),
```
Basically, how would it know when you are talking about column names vs variable(object)? Surround it by `!!` (which...apparently is said out loud "bang-bang")

Reply:
ah thanks! bang-bang! 

```!!(thanks)```

Reply:
woot! I forgot about `!!` and would have suggested just renaming `ref_date`. @ldecicco-USGS 's suggestion is way better.